manifests/0000_50_cluster-monitoring-operator_02-role.yaml

```diff
@@ -198,18 +198,6 @@ rules:
   - watch
   - patch
   - update
-- apiGroups:
-  - authentication.k8s.io
-  resources:
-  - tokenreviews
```

Comment:
I wonder why this cluster role loses ability to create token reviews and subject access reviews?

Reply:
the role was needed for grafana's oauth-proxy to validate tokens.

Reply:
and to grant the permissions to the grafana service account, CMO needed the same permissions (e.g. you can't grant something that you're not granted yourself).